README.md

```diff
@@ -1,2 +1,31 @@
-# hw01
-Repository for homework 01
+# Jerry Shi
+![Me in DC over Spring Break! I had a great time there and gained at least five pounds from all the junk food I ate.](jerry.jpg)
+
+Intro
+=====
+
+Hi there! My name is Jerry Shi and I'm currently a Second Year Business Economics major at the University of Chicago. I'm writing this readme for the very first assignment of the class ["Computing for the Social Sciences"](https://cfss.uchicago.edu). In my spare time, some of my hobbies include
+
+ * crew
+ * yoga
+ * going to live concerts
+ * exploring downtown (which I *definitely* do not do enough!)
+ 
+A little more about me
+----------------------
+
+1. Hometown?
+
+I grew up all over the place! Born in Reno, then lived in SoCal (LA & San Diego) through middle school, then went to high school in Indiana.
+
+2. Favorite food?
+
+Thai rolled ice cream
+
+3. Favorite genre of music?
+
+Pop all the way
+
+4. Fun fact?
+
+Chinese was my first language since I lived with my grandparents in China as a baby! Now English is the language I'm better at though, having grown up in the US.
```

Comment:
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       x    |              |            |
| **Coding strategy**         |      x     |              |            |
| **Presentation**            |    x       |              |            |
| **Achievement, creativity** |    x       |              |            |
| **Ease of access**          |      x     |              |            |

I think that this is an excellent piece of work!  I did not know how to do the subtitles that you used here, but now after seeing your example, I do.  I enjoyed reading your creative bio and appreciated your detailed comments about your process.